README.md

```diff
@@ -10,38 +10,39 @@ The technical specification of the data interface described in this document pro
 4. Function structure of the interface
 5. Types of communication with fiskaltrust.Middleware
 6. Operating categories
-Interface specification is provided by fiskaltrust.Interface nuget package, which can be found at https://www.nuget.org/packages/fiskaltrust.interface
+Interface specification is provided by fiskaltrust.Interface NuGet package, which can be found at https://www.nuget.org/packages/fiskaltrust.interface
 
 Official PDF documents that are built from the source in this repository can be found on the [Releases page](https://github.com/fiskaltrust/interface-doc/releases).
 
 ## Contributions
-If you want to contribute to this documentation, please review this README file to understand how it is structured and which tools are used.
+Please review this README file to understand how it is structured and which tools are used if you want to contribute to this documentation.
```

Comment:
By reviewing this README file in order to understand its structure and which tools are being used, you are contributing to improving this documentation.

Reply:
this is genius, thank you!